browserApp/app.py

```diff
@@ -43,22 +48,20 @@ def __init__(self, top_item, parent=None):
 
         self.show()
 
-
     def itemSelected(self, model_info):
         """ User as clicked on an item in the tree, pass item's data to the info view """
         self.info.populate(model_info)
         self.collection.populate(model_info)
 
 
-
-
 def show_app(top_item):
     app = QtWidgets.QApplication(sys.argv)
     win = App(top_item)
     app.exec_()  # blocking call
 
 
 if __name__ == "__main__":
-    top_item = model_file.FileItem("/Volumes/T7/GhostKid")
+    # top_item = model_file.FileItem("/Volumes/T7/GhostKid")
+    top_item = model_file.FileItem("D:\Programming\AlsSchool_Backup\class_3")
```

Comment:
it could be good to do this:
```
filename = None
for f in [ "/Volumes/...", "D:\aaaaa" ]:
   if os.path.isfile(f):
        filename = f
        break
if filename is None:
    raise RuntimeError("Could not find a path to test with!")
```